time stamp
int8 speed
float32 steeringAngle
